cp3pt0-deployment/migrate_tenant.sh

```diff
@@ -69,9 +69,6 @@ function main() {
     # Migrate singleton services
     if [[ $ENABLE_LICENSING -eq 1 ]]; then
         arguments+="--enable-licensing"
-        if [[ "$CONTROL_NS" != "$OPERATOR_NS" ]]; then
-            arguments+=" -licensingNs $CONTROL_NS"
-        fi
```

Comment:
Handle error: https://github.ibm.com/ibmprivatecloud/roadmap/issues/60056#issuecomment-61928346
It asks `setup_singleton.sh` to determine the namespace for Licensing Service.
```
[✘] An ibm-licensing-operator already installed in namespace: cs-control, expected namespace is: ibm-licensing
```